### ISSUE TITLE
Classification scheme won't load because of unrecognized field "type"

### ISSUE THREAD

**Issue by JelleKleevensVAI** [keeps/roda-in#372]
1. Download a classification scheme from RODA.
2. Load classification scheme in RODA-In (2.3.1).

> Error:
> 
> 2021-10-19 15:28:01 ERROR o.r.rodain.ui.schema.ui.SchemaPane - Error reading classification scheme specification
> com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "type" (class org.roda.rodain.core.schema.Sip), not marked as ignorable (7 known properties: "updateSIP", "parentId", "title", "additionalProperties", "id", "metadata", "descriptionlevel"])

The "type" field in the classification scheme seems to cause the issue. When I removed the field, the classification scheme could load in RODA-In.

**Comment by JelleKleevensVAI** [comment]
Hey guys, any chance this will be fixed in the next release?

**Comment by JoaoGomes2110** [comment]
Hi Jelle, will look into the issue and try to include it in a future release. I'll let you know when it's solved

**Comment by JoaoGomes2110** [comment]
Hi Jelle, can you give me the classification schema that you use ?

**Comment by JelleKleevensVAI** [comment]
This one for example. 

```json
{
	"dos": [
		{
			"title": "Bulk AIPs",
			"id": "4085e318-9e0f-4fa2-9f3b-945ade8d2afc",
			"parentId": "6c63fb22-ca07-4912-b559-6bc127aa7e1b",
			"descriptionlevel": "class",
			"type": "MIXED",
			"metadata": [
				{
					"id": "ead_2002.xml",
					"metadataType": "ead",
					"metadataVersion": "2002",
					"content": "PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4KPGVhZCB4bWxucz0idXJuOmlzYm46MS05MzE2NjYtMjItOSIgeG1sbnM6eHNpPSJodHRwOi8vd3d3LnczLm9yZy8yMDAxL1hNTFNjaGVtYS1pbnN0YW5jZSIgeHNpOnNjaGVtYUxvY2F0aW9uPSJ1cm46aXNibjoxLTkzMTY2Ni0yMi05IC4uL3NjaGVtYXMvZWFkMjAwMi54c2QiPgoKICA8ZWFkaGVhZGVyIGxhbmdlbmNvZGluZz0iaXNvNjM5LTJiIiBjb3VudHJ5ZW5jb2Rpbmc9ImlzbzMxNjYtMSIgZGF0ZWVuY29kaW5nPSJpc284NjAxIiByZXBvc2l0b3J5ZW5jb2Rpbmc9ImlzbzE1NTExIiBzY3JpcHRlbmNvZGluZz0iaXNvMTU5MjQiIHJlbGF0ZWRlbmNvZGluZz0iREMiPgogICAgPGVhZGlkIGVuY29kaW5nYW5hbG9nPSJpZGVudGlmaWVyIj40MDg1ZTMxOC05ZTBmLTRmYTItOWYzYi05NDVhZGU4ZDJhZmM8L2VhZGlkPgogICAgPGZpbGVkZXNjPgogICAgICA8dGl0bGVzdG10PgogICAgICAgIDx0aXRsZXByb3BlciBlbmNvZGluZ2FuYWxvZz0idGl0bGUiPjwvdGl0bGVwcm9wZXI+CiAgICAgIDwvdGl0bGVzdG10PgogICAgPC9maWxlZGVzYz4KICAgICAgICA8cHJvZmlsZWRlc2M+CiAgICAgICAgICAgIDxjcmVhdGlvbj5HZW5lcmF0ZWQgYnkgUk9EQSAtIFJlcG9zaXRvcnkgb2YgQXV0aGVudGljYSBEaWdpdGFsIE9iamVjdHM8L2NyZWF0aW9uPjwvcHJvZmlsZWRlc2M+CiAgICA8L2VhZGhlYWRlcj4KCiAgICA8YXJjaGRlc2MgbGV2ZWw9J2NsYXNzJyA+CiAgICAgICAgPGRpZD4KCiAgICAgICAgICAgIDx1bml0dGl0bGUgZW5jb2RpbmdhbmFsb2c9IjMuMS4yIiA+QnVsayBBSVBzPC91bml0dGl0bGU+PHVuaXRpZCBlbmNvZGluZ2FuYWxvZz0iMy4xLjEiICA+NDA4NWUzMTgtOWUwZi00ZmEyLTlmM2ItOTQ1YWRlOGQyYWZjPC91bml0aWQ+PGxhbmdtYXRlcmlhbD48bGFuZ3VhZ2U+RW5nbGlzaDwvbGFuZ3VhZ2U+RW5nbGlzaAogICAgICAgICAgICA8L2xhbmdtYXRlcmlhbD48L2RpZD4KICAgICAgICAKCgoKCiAgICAgICAgPG9kZCB0eXBlPSJsZXZlbE9mRGV0YWlsIj4KICAgICAgICAgICAgICAgIDxwPm1pbmltYWw8L3A+CiAgICAgICAgICAgIDwvb2RkPjxvZGQgdHlwZT0ic3RhdHVzRGVzY3JpcHRpb24iPgogICAgICAgICAgICAgICAgPHA+ZHJhZnQ8L3A+CiAgICAgICAgICAgIDwvb2RkPjxwcm9jZXNzaW5mbz48cD48ZGF0ZT4yMDIyLTAxLTIxPC9kYXRlPjwvcD48cD48L3A+CiAgICAgICAgICAgIDwvcHJvY2Vzc2luZm8+PGRzYyB0eXBlPSJjb21iaW5lZCI+CiAgICAgICAgPC9kc2M+CgogICAgPC9hcmNoZGVzYz4KPC9lYWQ+Cg==",
					"contentEncoding": "Base64"
				}
			]
		},
		{
			"title": "VAi",
			"id": "6c63fb22-ca07-4912-b559-6bc127aa7e1b",
			"descriptionlevel": "fonds",
			"type": "MIXED",
			"metadata": [
				{
					"id": "dc_SimpleDC20021212.xml",
					"metadataType": "dc",
					"metadataVersion": "SimpleDC20021212",
					"content": "PD94bWwgdmVyc2lvbj0iMS4wIj8+CjxzaW1wbGVkYyB4bWxuczp4c2k9Imh0dHA6Ly93d3cudzMub3JnLzIwMDEvWE1MU2NoZW1hLWluc3RhbmNlIgoJICB4c2k6bm9OYW1lc3BhY2VTY2hlbWFMb2NhdGlvbj0iLi4vc2NoZW1hcy9kYy54c2QiPjx0aXRsZT5WQWk8L3RpdGxlPjxpZGVudGlmaWVyPjZjNjNmYjIyLWNhMDctNDkxMi1iNTU5LTZiYzEyN2FhN2UxYjwvaWRlbnRpZmllcj48bGFuZ3VhZ2U+RW5nbGlzaDwvbGFuZ3VhZ2U+PHR5cGU+Rm9uZHM8L3R5cGU+PC9zaW1wbGVkYz4=",
					"contentEncoding": "Base64"
				}
			]
		},
		{
			"title": "Filebased AIPs",
			"id": "cc5dbcc3-786e-4dea-b0c7-13849a1128fc",
			"parentId": "6c63fb22-ca07-4912-b559-6bc127aa7e1b",
			"descriptionlevel": "class",
			"type": "MIXED",
			"metadata": [
				{
					"id": "ead_2002.xml",
					"metadataType": "ead",
					"metadataVersion": "2002",
					"content": "PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4KPGVhZCB4bWxucz0idXJuOmlzYm46MS05MzE2NjYtMjItOSIgeG1sbnM6eHNpPSJodHRwOi8vd3d3LnczLm9yZy8yMDAxL1hNTFNjaGVtYS1pbnN0YW5jZSIgeHNpOnNjaGVtYUxvY2F0aW9uPSJ1cm46aXNibjoxLTkzMTY2Ni0yMi05IC4uL3NjaGVtYXMvZWFkMjAwMi54c2QiPgoKICA8ZWFkaGVhZGVyIGxhbmdlbmNvZGluZz0iaXNvNjM5LTJiIiBjb3VudHJ5ZW5jb2Rpbmc9ImlzbzMxNjYtMSIgZGF0ZWVuY29kaW5nPSJpc284NjAxIiByZXBvc2l0b3J5ZW5jb2Rpbmc9ImlzbzE1NTExIiBzY3JpcHRlbmNvZGluZz0iaXNvMTU5MjQiIHJlbGF0ZWRlbmNvZGluZz0iREMiPgogICAgPGVhZGlkIGVuY29kaW5nYW5hbG9nPSJpZGVudGlmaWVyIj5jYzVkYmNjMy03ODZlLTRkZWEtYjBjNy0xMzg0OWExMTI4ZmM8L2VhZGlkPgogICAgPGZpbGVkZXNjPgogICAgICA8dGl0bGVzdG10PgogICAgICAgIDx0aXRsZXByb3BlciBlbmNvZGluZ2FuYWxvZz0idGl0bGUiPjwvdGl0bGVwcm9wZXI+CiAgICAgIDwvdGl0bGVzdG10PgogICAgPC9maWxlZGVzYz4KICAgICAgICA8cHJvZmlsZWRlc2M+CiAgICAgICAgICAgIDxjcmVhdGlvbj5HZW5lcmF0ZWQgYnkgUk9EQSAtIFJlcG9zaXRvcnkgb2YgQXV0aGVudGljYSBEaWdpdGFsIE9iamVjdHM8L2NyZWF0aW9uPjwvcHJvZmlsZWRlc2M+CiAgICA8L2VhZGhlYWRlcj4KCiAgICA8YXJjaGRlc2MgbGV2ZWw9J2NsYXNzJyA+CiAgICAgICAgPGRpZD4KCiAgICAgICAgICAgIDx1bml0dGl0bGUgZW5jb2RpbmdhbmFsb2c9IjMuMS4yIiA+RmlsZWJhc2VkIEFJUHM8L3VuaXR0aXRsZT48dW5pdGlkIGVuY29kaW5nYW5hbG9nPSIzLjEuMSIgID5jYzVkYmNjMy03ODZlLTRkZWEtYjBjNy0xMzg0OWExMTI4ZmM8L3VuaXRpZD48bGFuZ21hdGVyaWFsPjxsYW5ndWFnZT5FbmdsaXNoPC9sYW5ndWFnZT5FbmdsaXNoCiAgICAgICAgICAgIDwvbGFuZ21hdGVyaWFsPjwvZGlkPgogICAgICAgIAoKCgoKICAgICAgICA8b2RkIHR5cGU9ImxldmVsT2ZEZXRhaWwiPgogICAgICAgICAgICAgICAgPHA+bWluaW1hbDwvcD4KICAgICAgICAgICAgPC9vZGQ+PG9kZCB0eXBlPSJzdGF0dXNEZXNjcmlwdGlvbiI+CiAgICAgICAgICAgICAgICA8cD5kcmFmdDwvcD4KICAgICAgICAgICAgPC9vZGQ+PHByb2Nlc3NpbmZvPjxwPjxkYXRlPjIwMjItMDEtMjE8L2RhdGU+PC9wPjxwPjwvcD4KICAgICAgICAgICAgPC9wcm9jZXNzaW5mbz48ZHNjIHR5cGU9ImNvbWJpbmVkIj4KICAgICAgICA8L2RzYz4KCiAgICA8L2FyY2hkZXNjPgo8L2VhZD4K",
					"contentEncoding": "Base64"
				}
			]
		}
	]
}
```

**Comment by JoaoGomes2110** [comment]
Thank you @JelleKleevensVAI

**Comment by JoaoGomes2110** [comment]
Hi @JelleKleevensVAI  , the problem is in classification plan download from RODA. I will fix in the RODA, but for now you can do a quick fix to load the classification plan in RODA-in like this:
```json
{
	"dos": [
		{
			"title": "Bulk AIPs",
			"id": "4085e318-9e0f-4fa2-9f3b-945ade8d2afc",
			"parentId": "6c63fb22-ca07-4912-b559-6bc127aa7e1b",
			"descriptionlevel": "class",
			"contentType": {
				"packageType": "UNKNOWN",
				"value": "MIXED"
			},
			"metadata": [
				{
					"id": "ead_2002.xml",
					"metadataType": "ead",
					"metadataVersion": "2002",
					"content": "PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4KPGVhZCB4bWxucz0idXJuOmlzYm46MS05MzE2NjYtMjItOSIgeG1sbnM6eHNpPSJodHRwOi8vd3d3LnczLm9yZy8yMDAxL1hNTFNjaGVtYS1pbnN0YW5jZSIgeHNpOnNjaGVtYUxvY2F0aW9uPSJ1cm46aXNibjoxLTkzMTY2Ni0yMi05IC4uL3NjaGVtYXMvZWFkMjAwMi54c2QiPgoKICA8ZWFkaGVhZGVyIGxhbmdlbmNvZGluZz0iaXNvNjM5LTJiIiBjb3VudHJ5ZW5jb2Rpbmc9ImlzbzMxNjYtMSIgZGF0ZWVuY29kaW5nPSJpc284NjAxIiByZXBvc2l0b3J5ZW5jb2Rpbmc9ImlzbzE1NTExIiBzY3JpcHRlbmNvZGluZz0iaXNvMTU5MjQiIHJlbGF0ZWRlbmNvZGluZz0iREMiPgogICAgPGVhZGlkIGVuY29kaW5nYW5hbG9nPSJpZGVudGlmaWVyIj40MDg1ZTMxOC05ZTBmLTRmYTItOWYzYi05NDVhZGU4ZDJhZmM8L2VhZGlkPgogICAgPGZpbGVkZXNjPgogICAgICA8dGl0bGVzdG10PgogICAgICAgIDx0aXRsZXByb3BlciBlbmNvZGluZ2FuYWxvZz0idGl0bGUiPjwvdGl0bGVwcm9wZXI+CiAgICAgIDwvdGl0bGVzdG10PgogICAgPC9maWxlZGVzYz4KICAgICAgICA8cHJvZmlsZWRlc2M+CiAgICAgICAgICAgIDxjcmVhdGlvbj5HZW5lcmF0ZWQgYnkgUk9EQSAtIFJlcG9zaXRvcnkgb2YgQXV0aGVudGljYSBEaWdpdGFsIE9iamVjdHM8L2NyZWF0aW9uPjwvcHJvZmlsZWRlc2M+CiAgICA8L2VhZGhlYWRlcj4KCiAgICA8YXJjaGRlc2MgbGV2ZWw9J2NsYXNzJyA+CiAgICAgICAgPGRpZD4KCiAgICAgICAgICAgIDx1bml0dGl0bGUgZW5jb2RpbmdhbmFsb2c9IjMuMS4yIiA+QnVsayBBSVBzPC91bml0dGl0bGU+PHVuaXRpZCBlbmNvZGluZ2FuYWxvZz0iMy4xLjEiICA+NDA4NWUzMTgtOWUwZi00ZmEyLTlmM2ItOTQ1YWRlOGQyYWZjPC91bml0aWQ+PGxhbmdtYXRlcmlhbD48bGFuZ3VhZ2U+RW5nbGlzaDwvbGFuZ3VhZ2U+RW5nbGlzaAogICAgICAgICAgICA8L2xhbmdtYXRlcmlhbD48L2RpZD4KICAgICAgICAKCgoKCiAgICAgICAgPG9kZCB0eXBlPSJsZXZlbE9mRGV0YWlsIj4KICAgICAgICAgICAgICAgIDxwPm1pbmltYWw8L3A+CiAgICAgICAgICAgIDwvb2RkPjxvZGQgdHlwZT0ic3RhdHVzRGVzY3JpcHRpb24iPgogICAgICAgICAgICAgICAgPHA+ZHJhZnQ8L3A+CiAgICAgICAgICAgIDwvb2RkPjxwcm9jZXNzaW5mbz48cD48ZGF0ZT4yMDIyLTAxLTIxPC9kYXRlPjwvcD48cD48L3A+CiAgICAgICAgICAgIDwvcHJvY2Vzc2luZm8+PGRzYyB0eXBlPSJjb21iaW5lZCI+CiAgICAgICAgPC9kc2M+CgogICAgPC9hcmNoZGVzYz4KPC9lYWQ+Cg==",
					"contentEncoding": "Base64"
				}
			]
		},
		{
			"title": "VAi",
			"id": "6c63fb22-ca07-4912-b559-6bc127aa7e1b",
			"descriptionlevel": "fonds",
			"contentType": {
				"packageType": "UNKNOWN",
				"value": "MIXED"
			},
			"metadata": [
				{
					"id": "dc_SimpleDC20021212.xml",
					"metadataType": "dc",
					"metadataVersion": "SimpleDC20021212",
					"content": "PD94bWwgdmVyc2lvbj0iMS4wIj8+CjxzaW1wbGVkYyB4bWxuczp4c2k9Imh0dHA6Ly93d3cudzMub3JnLzIwMDEvWE1MU2NoZW1hLWluc3RhbmNlIgoJICB4c2k6bm9OYW1lc3BhY2VTY2hlbWFMb2NhdGlvbj0iLi4vc2NoZW1hcy9kYy54c2QiPjx0aXRsZT5WQWk8L3RpdGxlPjxpZGVudGlmaWVyPjZjNjNmYjIyLWNhMDctNDkxMi1iNTU5LTZiYzEyN2FhN2UxYjwvaWRlbnRpZmllcj48bGFuZ3VhZ2U+RW5nbGlzaDwvbGFuZ3VhZ2U+PHR5cGU+Rm9uZHM8L3R5cGU+PC9zaW1wbGVkYz4=",
					"contentEncoding": "Base64"
				}
			]
		},
		{
			"title": "Filebased AIPs",
			"id": "cc5dbcc3-786e-4dea-b0c7-13849a1128fc",
			"parentId": "6c63fb22-ca07-4912-b559-6bc127aa7e1b",
			"descriptionlevel": "class",
			"contentType": {
				"packageType": "UNKNOWN",
				"value": "MIXED"
			},
			"metadata": [
				{
					"id": "ead_2002.xml",
					"metadataType": "ead",
					"metadataVersion": "2002",
					"content": "PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4KPGVhZCB4bWxucz0idXJuOmlzYm46MS05MzE2NjYtMjItOSIgeG1sbnM6eHNpPSJodHRwOi8vd3d3LnczLm9yZy8yMDAxL1hNTFNjaGVtYS1pbnN0YW5jZSIgeHNpOnNjaGVtYUxvY2F0aW9uPSJ1cm46aXNibjoxLTkzMTY2Ni0yMi05IC4uL3NjaGVtYXMvZWFkMjAwMi54c2QiPgoKICA8ZWFkaGVhZGVyIGxhbmdlbmNvZGluZz0iaXNvNjM5LTJiIiBjb3VudHJ5ZW5jb2Rpbmc9ImlzbzMxNjYtMSIgZGF0ZWVuY29kaW5nPSJpc284NjAxIiByZXBvc2l0b3J5ZW5jb2Rpbmc9ImlzbzE1NTExIiBzY3JpcHRlbmNvZGluZz0iaXNvMTU5MjQiIHJlbGF0ZWRlbmNvZGluZz0iREMiPgogICAgPGVhZGlkIGVuY29kaW5nYW5hbG9nPSJpZGVudGlmaWVyIj5jYzVkYmNjMy03ODZlLTRkZWEtYjBjNy0xMzg0OWExMTI4ZmM8L2VhZGlkPgogICAgPGZpbGVkZXNjPgogICAgICA8dGl0bGVzdG10PgogICAgICAgIDx0aXRsZXByb3BlciBlbmNvZGluZ2FuYWxvZz0idGl0bGUiPjwvdGl0bGVwcm9wZXI+CiAgICAgIDwvdGl0bGVzdG10PgogICAgPC9maWxlZGVzYz4KICAgICAgICA8cHJvZmlsZWRlc2M+CiAgICAgICAgICAgIDxjcmVhdGlvbj5HZW5lcmF0ZWQgYnkgUk9EQSAtIFJlcG9zaXRvcnkgb2YgQXV0aGVudGljYSBEaWdpdGFsIE9iamVjdHM8L2NyZWF0aW9uPjwvcHJvZmlsZWRlc2M+CiAgICA8L2VhZGhlYWRlcj4KCiAgICA8YXJjaGRlc2MgbGV2ZWw9J2NsYXNzJyA+CiAgICAgICAgPGRpZD4KCiAgICAgICAgICAgIDx1bml0dGl0bGUgZW5jb2RpbmdhbmFsb2c9IjMuMS4yIiA+RmlsZWJhc2VkIEFJUHM8L3VuaXR0aXRsZT48dW5pdGlkIGVuY29kaW5nYW5hbG9nPSIzLjEuMSIgID5jYzVkYmNjMy03ODZlLTRkZWEtYjBjNy0xMzg0OWExMTI4ZmM8L3VuaXRpZD48bGFuZ21hdGVyaWFsPjxsYW5ndWFnZT5FbmdsaXNoPC9sYW5ndWFnZT5FbmdsaXNoCiAgICAgICAgICAgIDwvbGFuZ21hdGVyaWFsPjwvZGlkPgogICAgICAgIAoKCgoKICAgICAgICA8b2RkIHR5cGU9ImxldmVsT2ZEZXRhaWwiPgogICAgICAgICAgICAgICAgPHA+bWluaW1hbDwvcD4KICAgICAgICAgICAgPC9vZGQ+PG9kZCB0eXBlPSJzdGF0dXNEZXNjcmlwdGlvbiI+CiAgICAgICAgICAgICAgICA8cD5kcmFmdDwvcD4KICAgICAgICAgICAgPC9vZGQ+PHByb2Nlc3NpbmZvPjxwPjxkYXRlPjIwMjItMDEtMjE8L2RhdGU+PC9wPjxwPjwvcD4KICAgICAgICAgICAgPC9wcm9jZXNzaW5mbz48ZHNjIHR5cGU9ImNvbWJpbmVkIj4KICAgICAgICA8L2RzYz4KCiAgICA8L2FyY2hkZXNjPgo8L2VhZD4K",
					"contentEncoding": "Base64"
				}
			]
		}
	]
}
```

**Comment by JoaoGomes2110** [comment]
Related https://github.com/keeps/roda/issues/1986

**Comment by hmiguim** [comment]
Fixed in RODA version 4.3.x onwards
